From: Sven Eckelmann <sven@narfation.org>
Date: Thu, 1 Jan 2026 00:28:31 +0100
Subject: Reduce ath79-generic image size

The OM2P and PA300 devices only allow up to 6400 KB of flash size - but
with both kernel and image aligned to 256MB. This size comes from the 3
JFFS2 erase blocks (3 * 256 KB) which need to be reserved.

Signed-off-by: Sven Eckelmann <sven@narfation.org>

diff --git a/target/linux/ath79/generic/target.mk b/target/linux/ath79/generic/target.mk
index 27c3365a296735326b76e345b479a0f1ef8474ff..d659083a3287e13b60919ad64381a5b5bd06c6c9 100644
--- a/target/linux/ath79/generic/target.mk
+++ b/target/linux/ath79/generic/target.mk
@@ -1,4 +1,5 @@
 BOARDNAME:=Generic
+FEATURES += small_flash
 
 DEFAULT_PACKAGES += wpad-basic-mbedtls
 
